src/server.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+#
+# server.py
+# Created by Stephen Brimhall on 1/23/17
+# Copyright (c) SST CTF and Stephen Brimhall. All Rights Reserved
+#
+# This file implements the `server` class for the chatd instant messaging system
+
+# Begin system imports
+import sys;
```

Comment:
No, I was importing it at first because I thought I might. I'm gonna go through and remove all the unused imports.